db/snapshot_impl.h

```diff
@@ -29,6 +29,8 @@ class SnapshotImpl : public Snapshot {
 
   virtual SequenceNumber GetSequenceNumber() const override { return number_; }
 
+  virtual int64_t GetSnapshotTime() const override { return unix_time_; }
```

Comment:
We will be exposing another API called `GetTimestamp()` to return the user-defined timestamp associated with a snapshot. To avoid confusion, I feel that we should rename this method to something like `GetUnixTime()` (https://en.wikipedia.org/wiki/Unix_time).

Reply:
Remove `virtual` and keep `override`, https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html.